### ISSUE TITLE
Fatal Application Error

### ISSUE THREAD

**Issue by jogolyjoo** [OpenTTD/OpenTTD#9064]
Hi All, 

I am having an issue where i can not get the game to run. Anyone have a fix for this issue, Only seems to happen after the latest update!

![image](https://user-images.githubusercontent.com/82823568/115250837-dc7b6880-a121-11eb-80f9-2daa20610c4d.png)

```
*** OpenTTD Crash Report ***

Crash at: Mon Apr 19 14:12:39 2021
In game date: 2095-05-19 (20)

Crash reason:
 Exception: C0000005
 Location:  000000013FFA4CAD
 Message:   <none>

OpenTTD version:
 Version:    1.11.1 (0)
 NewGRF ver: 1b086d64
 Bits:       64
 Endian:     little
 Dedicated:  no
 Build date: Apr 18 2021 21:16:51

Registers:
 RAX: 00000000FF000000 RBX: 0000000000100800 RCX: 0000000000100800 RDX: 0000000000000000
 RSI: 0000000000000300 RDI: 0000000000000000 RBP: 0000000000000556 RSP: 00000000001DF170
 R8:  0000000000000000 R9:  00000000000088E8 R10: 0000000000000000 R11: 000000000347EEB0
 R12: 0000000000100800 R13: 0000000000000558 R14: 0000000000000020 R15: 0000000000402000
 RIP: 000000013FFA4CAD EFLAGS: 00010206

 Bytes at instruction pointer:
 F3 AB B9 EC 88 00 00 48 8B 5C 24 30 48 83 C4 20 5F 48 FF 25 DB 64 E4 01

Stack trace:
 00000300 00000000 00100800 00000000 00000556 00000000 00100800 00000000
 0333DFF0 00000000 3FFA17FB 00000001 04AD7C70 00000000 00000556 00000000
 04AD7C70 00000000 0333DFF0 00000000 4039FE10 00000001 02030410 00000000
 033458E0 00000000 0333DFF0 00000000 4039FE50 00000001 0047D7C0 00000000
 4039FE50 00000001 0047D7C0 00000000 00000300 00000000 00000001 00000000
 00000556 00000000 3FFA9F61 00000001 0333DFF0 00000000 02030410 00000000
 4039FE10 00000001 FF000000 00000000 40389028 00000001 3FFA8BCD 00000001
 FFFFFFFF FFFFFFFF 40388210 00000001 0333DFF0 00000000 3FFAB5BD 00000001
 00000001 00000000 3FFA9EA9 00000001 0333E020 00000000 00000000 00000000
 4039FE10 00000001 40389028 00000001 40389028 00000001 3FFAC0AC 00000001
 FFFFFFFF FFFFFFFF 3FFAC648 00000001 FFFFFFFF FFFFFFFF 40388210 00000001
 00000000 00000000 70623034 6E612D70 00006D69 00000000 0000000A 00000000
 0000000F 00000000 513C3CFA 000091D1 FFFFFFFF FFFFFFFF 4039FE10 00000001
 00000000 00000000 0333E030 00000000 0333DFF0 00000000 3FFAC3BF 00000001
 EA561A3F 00000337 001DF490 00000000 0333E080 00000000 001DF490 00000000
 00000000 00000000 3FFA8B9F 00000001 0333E080 00000000 0333E030 00000000
 0333DFF0 00000000 00000000 00000000 03345800 00000000 400E9CD9 00000001
 03345800 00000000 03345800 00000000 00000000 00000000 00000000 00000000
 000003F0 00000000 513C3D6A 000091D1 001DF340 00000000 00000001 00000000
 0000000A 00000000 001DF320 00000000 00000000 00000001 00000000 00000000
 00000000 00000000 00000000 00000000 00000000 000007FE 001DF888 00000000
 4035F350 00000001 00000000 00000000 00000000 00000000 001DF880 00000000
 00000000 00000000 3FF15D74 00000001 0000027F 00000000 00000000 00000000
 00000000 00000000 00001FA0 0000FFFF 00000000 00000000 00000000 00000000

Operating system:
 Name:     Windows
 Release:  6.1.7601 (Service Pack 1)
 Compiler: MSVC 1928

Configuration:
 Blitter:      40bpp-anim
 Graphics set: OpenGFX (7761)
 Language:     C:\Program Files (x86)\Steam\steamapps\common\OpenTTD\lang\english.lng
 Music driver: dmusic
 Music set:    OpenMSX (7761)
 Network:      no
 Sound driver: win32
 Sound set:    OpenSFX (7758)
 Video driver: win32-opengl

Fonts:
 Small:  sprite
 Medium: sprite
 Large:  sprite
 Mono:   sprite

AI Configuration (local: 0) (current: 0):
  0: Human
  1: Human
  2: Human
  3: Human
  4: Human
  5: Human

Libraries:
 LZMA:       5.2.5
 LZO:        2.10
 PNG:        1.6.37
 Zlib:       1.2.11

Module information:
 C:\Program Files (x86)\Steam\steamapps\common\OpenTTD\openttd.exe handle: 000000013FD20000 size: 8553984 crc: 9A796EBF date: 2021-04-19 12:05:09
 C:\Windows\SYSTEM32\ntdll.dll handle: 0000000077720000 size: 1671296 crc: 12C7AA90 date: 2020-01-03 03:35:05
 C:\Windows\system32\kernel32.dll handle: 0000000077500000 size: 1162752 crc: 52B7958F date: 2020-01-03 03:33:39
 C:\Windows\system32\KERNELBASE.dll handle: 000007FEFD2B0000 size: 408576 crc: 6945BC8E date: 2020-01-03 03:33:39
 C:\Windows\system32\OPENGL32.dll handle: 000007FED8260000 size: 1039872 crc: 821E408D date: 2009-07-14 01:41:53
 C:\Windows\system32\msvcrt.dll handle: 000007FEFE110000 size: 634880 crc: 9D1B448A date: 2011-12-16 08:46:06
 C:\Windows\system32\ADVAPI32.dll handle: 000007FEFE8C0000 size: 880640 crc: 796C4478 date: 2020-01-03 03:33:10
 C:\Windows\SYSTEM32\sechost.dll handle: 000007FEFD970000 size: 113664 crc: 98A6ADF4 date: 2015-05-25 18:19:09
 C:\Windows\system32\RPCRT4.dll handle: 000007FEFD840000 size: 1211392 crc: 4CAF040C date: 2020-01-03 03:33:53
 C:\Windows\system32\GDI32.dll handle: 000007FEFE810000 size: 405504 crc: C3BFCE30 date: 2019-11-28 03:28:53
 C:\Windows\system32\USER32.dll handle: 0000000077620000 size: 1010688 crc: 0EAFC823 date: 2020-01-03 03:33:57
 C:\Windows\system32\LPK.dll handle: 000007FEFE880000 size: 41472 crc: 4257860A date: 2019-11-15 02:21:52
 C:\Windows\system32\USP10.dll handle: 000007FEFEAF0000 size: 806400 crc: 8CB0A4ED date: 2019-11-05 21:20:28
 C:\Windows\system32\GLU32.dll handle: 000007FEEFB50000 size: 165376 crc: 3AEC0CD2 date: 2009-07-14 01:40:56
 C:\Windows\system32\DDRAW.dll handle: 000007FED83A0000 size: 569344 crc: 536DA9DD date: 2009-07-14 01:40:28
 C:\Windows\system32\DCIMAN32.dll handle: 000007FEFA7B0000 size: 14336 crc: 90953396 date: 2019-11-15 02:21:42
 C:\Windows\system32\SETUPAPI.dll handle: 000007FEFDD30000 size: 1900544 crc: A364BD6A date: 2010-11-21 03:24:28
 C:\Windows\system32\CFGMGR32.dll handle: 000007FEFD5F0000 size: 207872 crc: 1984C339 date: 2010-11-21 03:24:00
 C:\Windows\system32\OLEAUT32.dll handle: 000007FEFE9B0000 size: 878080 crc: 9F34D3E8 date: 2019-11-15 02:22:04
 C:\Windows\system32\ole32.dll handle: 000007FEFDF10000 size: 2072576 crc: 05ECA873 date: 2019-11-15 02:22:04
 C:\Windows\system32\DEVOBJ.dll handle: 000007FEFD500000 size: 93184 crc: A7541AE2 date: 2009-07-14 01:40:28
 C:\Windows\system32\dwmapi.dll handle: 000007FEFB520000 size: 82944 crc: F979E48B date: 2015-07-09 17:58:26
 C:\Windows\system32\WS2_32.dll handle: 000007FEFD990000 size: 296448 crc: EE1BC529 date: 2016-05-11 17:02:50
 C:\Windows\system32\NSI.dll handle: 000007FEFE9A0000 size: 13312 crc: C65F00AA date: 2017-08-11 06:35:01
 C:\Windows\system32\WINMM.dll handle: 000007FEFB170000 size: 217600 crc: 7BC1529F date: 2009-07-14 01:41:56
 C:\Windows\system32\IMM32.dll handle: 000007FEFE890000 size: 167424 crc: 41B2805A date: 2009-07-14 01:41:09
 C:\Windows\system32\MSCTF.dll handle: 000007FEFE660000 size: 1078784 crc: E5722AE3 date: 2019-08-29 02:50:36
 C:\Windows\system32\SHELL32.dll handle: 000007FEFEBC0000 size: 14185984 crc: BFB2A553 date: 2019-05-25 00:04:16
 C:\Windows\system32\SHLWAPI.dll handle: 000007FEFD7C0000 size: 448512 crc: 086DF743 date: 2010-11-21 03:24:22
 C:\Program Files (x86)\Steam\gameoverlayrenderer64.dll handle: 000007FED6130000 size: 1809640 crc: 8A9EFDDF date: 2021-04-12 20:03:30
 C:\Windows\system32\PSAPI.DLL handle: 00000000778D0000 size: 9216 crc: 84B5E9BA date: 2009-07-14 01:41:53
 C:\Windows\system32\api-ms-win-core-synch-l1-2-0.DLL handle: 000007FEF8200000 size: 12024 crc: F44C9C18 date: 2019-04-12 13:05:06
 C:\Windows\system32\uxtheme.dll handle: 000007FEFB9E0000 size: 332288 crc: B0A13DF9 date: 2009-07-14 01:41:56
 C:\Windows\system32\ig7icd64.dll handle: 0000000180000000 size: 17226240 crc: B6302B4A date: 2012-09-09 21:15:27
 C:\Windows\system32\CRYPTBASE.dll handle: 000007FEFD110000 size: 43520 crc: 739E6DEC date: 2020-01-03 03:33:20
 C:\Windows\system32\CLBCatQ.DLL handle: 000007FEFE770000 size: 607744 crc: 939E35C9 date: 2009-07-14 01:40:15
 C:\Windows\System32\MMDevApi.dll handle: 000007FEFB940000 size: 284160 crc: CDF2EF81 date: 2009-07-14 01:41:26
 C:\Windows\System32\PROPSYS.dll handle: 000007FEFBAF0000 size: 1212416 crc: B83FA3BE date: 2010-11-21 03:23:55
 C:\Windows\system32\AUDIOSES.DLL handle: 000007FEFB040000 size: 295936 crc: B227EAC3 date: 2019-09-12 03:44:27
 C:\Windows\system32\mswsock.dll handle: 000007FEFC9F0000 size: 327168 crc: 2E4865A8 date: 2016-05-11 17:02:42
 C:\Windows\System32\wshtcpip.dll handle: 000007FEFC3D0000 size: 13312 crc: 5D1E2C55 date: 2009-07-14 01:41:58
 C:\Windows\system32\wdmaud.drv handle: 000007FEFAB70000 size: 217088 crc: 12829973 date: 2010-11-21 03:24:32
 C:\Windows\system32\ksuser.dll handle: 0000000074CE0000 size: 5120 crc: 781141DD date: 2015-12-08 19:07:31
 C:\Windows\system32\AVRT.dll handle: 000007FEFB930000 size: 18432 crc: 5CE53CB7 date: 2009-07-14 01:40:05
 C:\Windows\system32\msacm32.drv handle: 000007FEFAA60000 size: 25600 crc: 2B579A05 date: 2009-07-14 01:38:50
 C:\Windows\system32\MSACM32.dll handle: 000007FEFAA40000 size: 83456 crc: 3D67538B date: 2009-07-14 01:41:27
 C:\Windows\system32\midimap.dll handle: 000007FEFAA30000 size: 20480 crc: CB902A13 date: 2009-07-14 01:41:23
 C:\Windows\System32\dmusic.dll handle: 000007FEE1CA0000 size: 125952 crc: 11947851 date: 2009-07-14 01:40:31
 C:\Windows\System32\DSOUND.dll handle: 000007FEFB1B0000 size: 540672 crc: B6225596 date: 2009-07-14 01:40:33
 C:\Windows\System32\POWRPROF.dll handle: 000007FEFB990000 size: 167424 crc: F6D04424 date: 2009-07-14 01:41:53
 C:\Windows\system32\WINTRUST.dll handle: 000007FEFD490000 size: 229376 crc: 4A3D489A date: 2019-12-10 08:32:47
 C:\Windows\system32\CRYPT32.dll handle: 000007FEFD320000 size: 1484800 crc: 8B142E5A date: 2019-12-10 08:32:28
 C:\Windows\system32\MSASN1.dll handle: 0000000004650000 size: 46592 crc: 50F9CBBE date: 2010-11-21 03:24:22
 C:\Windows\System32\dmsynth.dll handle: 000007FED8160000 size: 119296 crc: 05A882A4 date: 2009-07-14 01:40:31

---- gamelog start ----
Tick 0: new game started
Revision text changed to 1.10.3, savegame version 218, not modified, _openttd_newgrf_version = 0x1a086d64
New game mode: 1 landscape: 2
Tick 1280: game loaded
New game mode: 2 landscape: 2
Tick 1280: game loaded
New game mode: 1 landscape: 2
Tick 39224: game loaded
New game mode: 2 landscape: 2
Tick 39224: game loaded
New game mode: 1 landscape: 2
Tick 60493: settings changed
Setting changed: vehicle.never_expire_vehicles : 0 -> 1
Tick 52147: game loaded
New game mode: 2 landscape: 2
Tick 52147: game loaded
New game mode: 1 landscape: 2
Tick 39493: game loaded
New game mode: 2 landscape: 2
Tick 39493: game loaded
New game mode: 1 landscape: 2
Tick 29222: game loaded
New game mode: 2 landscape: 2
Tick 29222: game loaded
New game mode: 1 landscape: 2
Tick 29222: game loaded
Revision text changed to 1.11.1, savegame version 290, not modified, _openttd_newgrf_version = 0x1b086d64
New game mode: 0 landscape: 2
---- gamelog end ----

Recent news messages:

*** End of OpenTTD Crash Report ***

Decoded stack trace:
[00] openttd              0x000000013FFA4CAD
[01] openttd              0x000000013FFA17FB
[02] openttd              0x000000013FFA9F61
[03] openttd              0x000000013FFA8BCD
[04] openttd              0x000000013FFA9EA9
[05] openttd              0x000000013FFAC0AC
[06] openttd              0x000000013FFAC3BF
[07] openttd              0x000000013FFA8B9F
[08] openttd              0x00000001400E9CD9
[09] openttd              0x000000013FF20CF8
[10] openttd              0x0000000140287E92
[11] kernel32             0x000000007751556D BaseThreadInitThunk + 13
[12] ntdll                0x000000007777372D RtlUserThreadStart + 29

*** End of additional info ***

```

**Comment by LordAro** [comment]
Can you also upload the various crash.* files located in `C:\Users\\\<user>\Documents\OpenTTD` (particularly crash.dmp) ?

Also details about your GPU hardware (and driver versions) would be greatly appreciated.

In the meantime, it's likely that this is the same issue as #8967 or #8969 , so try disabling the hardware acceleration setting in openttd.cfg as mentioned in the issue (you can open it in Notepad)

**Comment by jogolyjoo** [comment]
Hi there,

thanks for your speedy reply!

I was totally broken when I couldn't start it this morning, playing the game again has brought me so much joy over the weekend! You are all amazing for keeping it going and improving it like you have! :)

**Comment by jogolyjoo** [comment]
Sorry I am having a hard time working out how to get this file uploaded on here, maybe its my old PC but it wont let me drag and drop or copy paste! I never felt so technologicall backwards until I found github!

**Comment by LordAro** [comment]
GitHub doesn't allow sav/dmp files, you'll need to put them in a zip

**Comment by jogolyjoo** [comment]
Yeah that's not working either, I tried the hardware acceleration solution, this worked but ended up with everything on screen totally zoomed in, on loading the game the world is ok, just all the top tiles are huge!

**Comment by jogolyjoo** [comment]
Aha think this worked

**Comment by jogolyjoo** [comment]
Let me know if you can see this zip

**Comment by FLHerne** [comment]
Not sure what you've done but, we cannot see the zip, sorry.

See [this page](https://docs.github.com/en/github/managing-your-work-on-github/file-attachments-on-issues-and-pull-requests) for how to attach a file to an issue or comment.

**Comment by Potato-cat1** [comment]
I had the same exact problem, let me send you the zip

**Comment by Potato-cat1** [comment]
Sorry, I don't know how to zip a file.

**Comment by PeterN** [comment]
> Sorry, I don't know how to zip a file.

Find file in explorer -> right click -> Send to -> Compressed (zipped) folder

**Comment by Potato-cat1** [comment]
[crash (2).zip](https://github.com/OpenTTD/OpenTTD/files/6337922/crash.2.zip)
[crash (2).zip](https://github.com/OpenTTD/OpenTTD/files/6337926/crash.2.zip)
 Did it work?

**Comment by glx22** [comment]
@Potato-cat1's one is a duplicate of #8969

**Comment by LordAro** [comment]
Duplicate of #8969. Should be fixed in 1.11.2, released now!
